tests/testthat/test-bump_version.R

```diff
@@ -0,0 +1,13 @@
+test_that("bump_version() works", {
+  tempdir <- withr::local_tempdir(pattern = "fledge")
+  repo <- create_demo_project(open = FALSE, dir = tempdir, news = TRUE)
+  usethis::with_project(
+    path = repo, {
+      use_r("bla")
+      gert::git_add("R/bla.R")
+      gert::git_commit("* Add cool bla.")
+      bump_version()
+    }
+  )
```

Comment:
Is it useful to record a snapshot of this code?

Reply:
What do you mean? Do you disagree with the file snapshot here or do you want more things to be in a snapshot?

Reply:
We could have both the file snapshot and the snapshot for the output, if it's stable.